### DISCUSSION TITLE
是否能够整理列表说明各个版本的区别，试用和付费区别

### DISCUSSION THREAD

**Discussion by hackdoors** [dashingsoft/pyarmor#1134] [Ideas]
发现该项目这么多issues，很多是由于用法不熟悉所致，文档复杂
以下是我的建议：
1、列表分别说明各个版本的用法对比，试用版本和付费版本用法哪些区别。
2、哪些是软件存在的bug，有没有修复，什么版本修复了
3、产品应该尽量简单易用，建议完善pyarmor-webui版本的详细教程，报错处理

**Comment by jondy** [comment]
谢谢建议，现在文档有每一个版本的修改日志（英文版），以及详细的许可协议，里面包含试用版本和付费版本的区别。

pyarmor-webui 基本就是把 pyarmor 命令行的选项转换成为图形界面，每一个选项下面有相关的说明。

**Comment by jondy** [comment]
这里是不同许可证和功能列表
https://pyarmor.readthedocs.io/zh/stable/licenses.html
